README.md

```diff
@@ -11,7 +11,7 @@ A module to store and interact with blocks.
 
 # EXAMPLE
 
-The following is an example to iterate through an existing Geth DB (needs ``leveldown`` to be
+The following is an example to iterate through an existing Geth DB (needs ```levelup``` and ``leveldown`` to be
```

Comment:
Currently, its recommended to use the convenience package ``level`` instead of ``levelup`` + ``leveldown``.
```suggestion
The following is an example to iterate through an existing Geth DB (needs ``level`` to be
```